src/riak_kv_http_cache.erl

```diff
@@ -40,15 +44,15 @@ code_change(_, S, _) ->
     {ok, S}.
 
 check_cache(#st{ts = undefined} = S) ->
-    S#st{ts = os:timestamp(), stats = do_get_stats()};
+    Stats = riak_kv_status:get_stats(web),
+    S#st{ts = os:timestamp(), stats = Stats};
 check_cache(#st{ts = Then} = S) ->
+    CacheTime = application:get_env(riak_kv, http_stats_cache_seconds, 1),
```

Comment:
One could be generous and make the configurable time in milliseconds instead of course seconds. That would help if one ever want to test these timeouts in QuickCheck like fashion.
Since this is a new parameter, setting default to 1000 and have it provided in milliseconds is not harming backward compatibility.

Reply:
Changed to milliseconds